docs/serverless/rules/rules-ui-management.mdx

```diff
@@ -111,6 +112,37 @@ When duplicating a rule with exceptions, you can choose to duplicate the rule an
     * To enable or disable a single rule, switch on the rule's **Enabled** toggle.
     * To <DocLink slug="/serverless/security/rules-ui-management" section="snooze-rule-actions">snooze</DocLink> actions for rules, click the bell icon.
 
+<div id="manually-run-rules"></div>
+
+## Run rules manually
+
+<DocBadge template="technical preview" />
+
+Manually run enabled rules for a specfied period of time for testing purposes or additional rule coverage.
+
+<DocCallOut title="Important" color="warning">
+Before manually running rules, make sure you properly understand and plan for rule dependencies. Incorrect scheduling can lead to inconsistent rule results.
+</DocCallOut>
+
+1. Go to **Rules** -> **Detection rules (SIEM)**.
+1. Do one of the following:
+    * Select the **All actions** menu (<DocIcon type="boxesHorizontal" title="Actions menu" size="m" />) on a rule, then select **Manual run**.
+    * Select all the rules you want to manually run, select the **Bulk actions** menu, then select **Manual run**.
+1. Specify when the manual run starts and ends. The default selection is the current day starting three hours in the past. The rule will search for events during the selected time range.
+1. Click **Run** to manually run the rule.
+   <DocCallOut title="Note">
+   Manual runs can produce multiple rule executions. This is determined by the manual run's time range and the rule's execution schedule.
+   </DocCallOut>
+
+The manual run's details are shown in the <DocLink slug="/serverless/security/alerts-ui-monitor" section="manual-runs-table">Manual runs</DocLink> table on the **Execution results** tab. Changes you make to the manual run or rule settings will display in the Manual runs table after the current run completes.
+
+<DocCallOut title="Note">
+Be mindful of the following:
+* Rules actions are not activated during manual runs. 
```

Comment:
We call them "rule actions" [elsewhere](https://elastic-dot-co-docs-production-12rpxe8p2-elastic-dev.vercel.app/current/serverless/security/rules-create#set-up-rule-actions-optional):
```suggestion
* Rule actions are not activated during manual runs. 
```